modules/nf-core/bowtie/align/main.nf

```diff
@@ -3,9 +3,10 @@ process BOWTIE_ALIGN {
     label 'process_high'
 
     conda "${moduleDir}/environment.yml"
+    container ""
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/mulled-v2-ffbf83a6b0ab6ec567a336cf349b80637135bca3:c84c7c55c45af231883d9ff4fe706ac44c479c36-0' :
-        'biocontainers/mulled-v2-ffbf83a6b0ab6ec567a336cf349b80637135bca3:c84c7c55c45af231883d9ff4fe706ac44c479c36-0' }"
+        'oras://community.wave.seqera.io/library/bowtie_samtools:16f00b34cfc72399' :
```

Comment:
I think this will break `nf-core download`. I'd prefer the https links if possible, and should test the download if we can.

Reply:
Is there a Seqera feedback link for the https links?